raw-spans-grouper/raw-spans-grouper/src/main/java/org/hypertrace/core/rawspansgrouper/RawSpansGrouper.java

```diff
@@ -59,18 +61,22 @@ public StreamsBuilder buildTopology(
 
     // Retrieve the default value serde defined in config and use it
     Serde valueSerde = defaultValueSerde(properties);
-    StoreBuilder<KeyValueStore<TraceIdentity, ValueAndTimestamp<RawSpans>>> traceStoreBuilder =
+    Serde keySerde = defaultKeySerde(properties);
+    StoreBuilder<WindowStore<SpanIdentity, RawSpan>> traceStoreBuilder =
+        Stores.windowStoreBuilder(
+            new RocksDbWindowBytesStoreSupplier(
+                SPAN_WINDOW_STORE,
+                Duration.ofHours(1).toMillis(),
```

Comment:
can u add a javadoc explaining the properties passed to the supplier and the behavior ?

Reply:
Added